src/calculateRentalCost.js

```diff
@@ -4,7 +4,16 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const dailyRate = 40;
+  let totalCost = days * dailyRate;
+
+  if (days >= 7) {
+    totalCost -= 50;
+  } else if (days >= 3) {
```

Comment:
[checklist](https://github.com/mate-academy/js_task-transportation-on-vacation/blob/master/checklist.md): Prefer `if` with `return` over `if else` to simplify later conditions.